### PR TITLE
Update README, Site-wide Settings and User Guide

### PR DESCRIPTION
**Pull request by chaiwanlin**
* Adapted README to EdRecord project.
* Updated site-wide settings to EdRecord project and repository.
* Updated Adding a student, Editing a student, Delete a student features in User Guide.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -73,42 +73,42 @@ Shows a message explaning how to access the help page.
...
-### Editing a person : `edit`
+### Editing a student : `edit`
 
-Edits an existing person in the address book.
+Edits an existing student in the record.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [m/MODULE] [c/CLASS] [i/INFO] [t/TAG]…​`
 
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
```
Comment (FergusMok) [inline docs/UserGuide.md:102]:
```suggestion
* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed student list. The index **must be a positive integer** 1, 2, 3, …​
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -128,19 +128,19 @@ Examples:
...
+### Deleting a student : `delete`
 
-Deletes the specified person from the address book.
+Deletes the specified student from the record.
 
 Format: `delete INDEX`
 
-* Deletes the person at the specified `INDEX`.
+* Deletes the student at the specified `INDEX`.
 * The index refers to the index number shown in the displayed person list.
```
Comment (FergusMok) [inline docs/UserGuide.md:138]:
```suggestion
* The index refers to the index number shown in the displayed student list.
```

File: docs/index.md [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
 ---
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2122S1-CS2103-W14-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103-W14-3/tp/actions)
 [![codecov](https://codecov.io/gh/AY2122S1-CS2103-W14-3/tp/branch/master/graph/badge.svg?token=5QC0BG91OQ)](https://codecov.io/gh/AY2122S1-CS2103-W14-3/tp)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**EdRecord is a desktop application for keeping track of your students.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
```
Comment (FergusMok) [inline docs/index.md:11]: Would it be better to sync with the current description we have: 
https://github.com/nus-cs2103-AY2122S1/tp/pull/11#issue-990816195

**Review by FergusMok — COMMENTED** [review]
Other than the small stuff, LGTM

File: docs/index.md (hunk 3 above)
Comment (chaiwanlin) [reply docs/index.md]: Adapted to your comment!

**Pull request closed without merging**